Library/Formula/osquery.rb

```diff
@@ -52,9 +53,9 @@ def install
   test do
     require 'open3'
     Open3.popen3("#{bin}/osqueryi") do |stdin, stdout, _|
-      stdin.write(".mode line\nSELECT major FROM osx_version;")
+      stdin.write(".mode line\nSELECT version FROM osquery_info;")
       stdin.close
-      assert_equal "major = 10\n", stdout.read
+      assert_equal "version = #{version}\n", stdout.read
```

Comment:
We got rid of the osx_version table. This is more of a legitimate test, I'm not sure I understand why this is a problem.